src/PHPUnitCommand.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Eris;
+
+final class PHPUnitCommand
+{
+    private $seed;
+    private $name;
+
+    private function __construct($seed, $name)
+    {
+        $this->seed = $seed;
+        $this->name = $name;
+    }
+
+    public static function fromSeedAndName($seed, $name)
+    {
+        return new self(
+            $seed,
+            str_replace('\\', '\\\\', $name)
+        );
+    }
+
+    public function __toString()
+    {
+        return "ERIS_SEED={$this->seed} vendor/bin/phpunit --filter {$this->name}";
```

Comment:
conceptually I think the name should be escaped here, because until you have it in memory you don't need to do it. But doesn't matter very much